protocol/app/app.go

```diff
@@ -675,10 +684,6 @@ func New(
 					)
 				}
 			}()
-			// Don't panic if metrics daemon loops are delayed. Use maximum value.
```

Comment:
The metrics daemon is such a simple go-routine that I don't think it makes sense to re-engineer it at this time. We'll just remove the previous health check. The task loop does not produce any errors because the telemetry calls don't produce errors either, so this daemon would essentially never become unhealthy.